packages/sync-actions/src/product-actions.js

```diff
@@ -41,6 +48,15 @@ export const referenceActionsList = [
  * HELPER FUNCTIONS
  */
 
+const addAction = (key, resource) =>
+  REGEX_NUMBER.test(key) && Array.isArray(resource) && resource.length
+
+const updateAction = (key, resource) =>
+  REGEX_NUMBER.test(key) && Object.keys(resource).length
+
+const removeAction = (key, resource) =>
```

Comment:
Nit: Still want to share it. Usually we (MC Team) tried to name functions which return booleans as getIs*. This has the advantage of communicating clearly what is returned while being assignable as isFoo = getIsFoo() without clashing names. 

Take it it leave it 🤡🤠